UI/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Welcome to iReporter</title>
+	<link rel="stylesheet" type="text/css" href="style/style.css">
+</head>
+<body>
+	<center>
+			<div class="land">
+				<h1>iReporters</h1>
+				<p>With the high rise of corruption and deploration of our country's infrastructures, it is time for the citizens to take charge of the situation. iReporter is a platform where citizens can share accounts of corruption witnessed or things that need urgent goverment intervention to the notice of appropriate authorities and the
+				general public.</p>
```

Comment:
the indenting here is too much and inconsistent. What Editor are you using?

Reply:
I'm using Sublime